### PR TITLE
Solution of py-cat-and-dog-years

### PR DESCRIPTION
**Pull request by AndriiHamasa**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,36 @@
 from app.main import get_human_age
 
-# write your code here
+
```
Comment (oliasymak24) [inline app/test_main.py:3]: Use @pytest.parametrize

Comment (oliasymak24) [reply app/test_main.py:3]: Add tests for negative numbers and incorrect types

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+        ),
+
+    ]
+)
+def test_get_human_age(
+        cat_age: int,
+        dog_age: int,
+        expected_age_list: list
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected_age_list
```
Comment (oliasymak24) [inline app/test_main.py:40]: Add one more test case if the function receives an incorrect type of data it raises the correct exception.

**Review by oliasymak24 — CHANGES_REQUESTED** [review]
